### DISCUSSION TITLE
how to change the status of release to draft state again on github

### DISCUSSION THREAD

**Discussion by adarsh-dell** [community/community#58749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how to change the status of release to draft state again on github?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @adarsh-dell , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
